includes/admin/views/html-admin-page-status-report.php

```diff
@@ -82,15 +85,28 @@
 		</tr>
 		<tr>
 			<td data-export-label="CMS Version"><?php esc_html_e( 'CMS version', 'woocommerce' ); ?>:</td>
-			<td class="help"><?php echo wc_help_tip( esc_html__( 'The CMS version installed on your site.', 'woocommerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
+			<td class="help"><?php echo wc_help_tip( esc_html__( 'The version of the CMS installed on your site.', 'woocommerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
 			<td>
 				<?php
-				if ( function_exists( 'classicpress_version' ) ) {
-				    /* translators: ClassicPress version string */
-				    printf( __( 'You are running ClassicPress version %s', 'woocommerce' ), classicpress_version() );
+				$latest_version = get_transient( 'woocommerce_system_status_wp_version_check' );
+
+				if ( false === $latest_version ) {
+					$version_check = wp_remote_get( 'https://api.wordpress.org/core/version-check/1.7/' );
+					$api_response  = json_decode( wp_remote_retrieve_body( $version_check ), true );
+
+					if ( $api_response && isset( $api_response['offers'], $api_response['offers'][0], $api_response['offers'][0]['version'] ) ) {
+						$latest_version = $api_response['offers'][0]['version'];
+					} else {
+						$latest_version = $environment['wp_version'];
+					}
+					set_transient( 'woocommerce_system_status_wp_version_check', $latest_version, DAY_IN_SECONDS );
+				}
+
+				if ( version_compare( $environment['wp_version'], $latest_version, '<' ) ) {
+					/* Translators: %1$s: Current version, %2$s: New version */
+					echo '<mark class="error"><span class="dashicons dashicons-warning"></span> ' . sprintf( esc_html__( '%1$s - There is a newer version of WordPress available (%2$s)', 'woocommerce' ), esc_html( $environment['wp_version'] ), esc_html( $latest_version ) ) . '</mark>';
 				} else {
-				    /* translators: WordPress version string */
-				    printf( __( 'You are running WordPress version %s', 'woocommerce' ), get_bloginfo( 'version' ) );
+					echo '<mark class="yes">' . esc_html( $environment['wp_version'] ) . '</mark>';
 				}
```

Comment:
This part still needs to be rewritten to include checks for CP version and WP versions. Ideally it would also give a warning if CP version is out of date, as it does for WP.